datastore/db/queries/inference/__init__.py

```diff
@@ -322,7 +373,27 @@ def set_inference_object_valid(cursor, inference_object_id: str, is_valid:bool):
         cursor.execute(query, (is_valid,inference_object_id))
     except Exception:
         raise Exception(f"Error: could not update valid for object {inference_object_id}")
+
+def check_inference_object_exist(cursor, inference_object_id):
+    """
+    Check if an inference object exists in the database.
     
+    return True if exists, False otherwise
+    """
+    try:
+        query = """
```

Comment:
This works but 
```suggestion
        query = ""SELECT EXISTS(SELECT 1 FROM ..."
```
handles the Boolean side without the uses of a None 👍🏻